linkis-spring-cloud-services/linkis-service-gateway/linkis-gateway-core/src/main/scala/org/apache/linkis/gateway/route/GatewayRouter.scala

```diff
@@ -129,11 +129,15 @@ abstract class AbstractGatewayRouter extends GatewayRouter with Logging {
 }
 
 class DefaultGatewayRouter(var gatewayRouters: Array[GatewayRouter]) extends AbstractGatewayRouter {
+  var nullRouters = gatewayRouters.filter(x => x == null)
```

Comment:
It is recommended to also make a non-empty judgment on the gatewayRouters collection. and take only the notNullRouters part

Reply:
ok